sql/entity_mapping.sql

```diff
@@ -0,0 +1,17 @@
+INSERT IGNORE INTO entity_mapping
+    (uuid,entity_mapping_type_id,entity1_uuid,entity2_uuid)
+VALUES
+    (uuid(),5,'d91e7c8a-0ebe-4792-a332-a175d881fe3a','df204697-a6c6-4768-a4e8-7082940fc8eb'),
```

Comment:
@prashantsable1991 Please use hard-coded uuid values; this will ensure we have the same uuid accross all our environments, and will also simplify future updates (by easily targetting a single record by uuid)

Reply:
@tmvumbi2 Thanks for the review. I will add all the suggested changes.